frontend/dockerfile/dockerfile_test.go

```diff
@@ -6723,10 +6757,12 @@ func testNamedImageContextPlatform(t *testing.T, sb integration.Sandbox) {
 	}, nil)
 	require.NoError(t, err)
 
-	dockerfile = []byte(`
+	dockerfile = []byte(integration.UnixOrWindows(`
 FROM --platform=$BUILDPLATFORM busybox AS target
-RUN echo hello
-`)
+RUN echo hello`,
```

Comment:
nit: can we maintain the same code styling/indentation as the rest, i.e
```go
        dockerfile = []byte(integration.UnixOrWindows(
            `
FROM ...
...
`,
            `
FROM ...
...
`,
        ))
```

Reply:
Done